src/diff-snapshot.js

```diff
@@ -181,6 +176,27 @@ function diffImageToSnapshot(options) {
   return result;
 }
 
+
+function runDiffImageToSnapshot(options) {
+  options.receivedImageBuffer = options.receivedImageBuffer.toString('base64');
+
+  const serializedInput = JSON.stringify(options);
+
+  let result = {};
+  const writeDiffProcess = childProcess.spawnSync('node', [`${__dirname}/write-result-diff-image.js`], { input: Buffer.from(serializedInput) });
+  if (writeDiffProcess.status === 0) {
+    result = JSON.parse(writeDiffProcess.stdout.toString());
+  }
+
+  if (writeDiffProcess.stderr.toString()) {
+    /* eslint-disable no-console */
+    console.log(writeDiffProcess.stderr.toString());
+  }
+
+  return result;
```

Comment:
What does it look like to the user if there is an issue in the child process while attempting to compare images and so `result` here ends up being an empty object? Curious about the messaging to avoid confusing the user.

Reply:
It doesn't look good 😄 We should probably wrap and throw stderr if the process exit status != 0.